src/emulator/modules/SceTouch/SceTouch.cpp

```diff
@@ -24,14 +24,95 @@
 
 // TODO Move elsewhere.
 static uint64_t timestamp;
+SDL_TouchFingerEvent finger_buffer[8];
+uint8_t finger_count = 0;
+long int finger_id_ref;
+auto touchscreen_port = SCE_TOUCH_PORT_FRONT;
+
+SceTouchData recover_touch_events() {
```

Comment:
For function that stays on this cpp file and not accessible on other cpp files, use ``static`` modifier

Reply:
so it should be static SceTouchData recover_touch_events() ?

Reply:
Yes that's correct. Google static linkage if you want to find out why i suggest it